frontend/public/components/namespace.jsx

```diff
@@ -530,25 +532,44 @@ const getProjectSelectedColumns = ({ showMetrics, showActions }) => {
 const ProjectLink = connect(null, {
   filterList: k8sActions.filterList,
 })(({ project, filterList }) => {
-  const [, setActiveNamespace] = useActiveNamespace();
+  const [, setLastNamespace] = useUserSettingsCompatibility(
+    LAST_NAMESPACE_NAME_USER_SETTINGS_KEY,
+    LAST_NAMESPACE_NAME_LOCAL_STORAGE_KEY,
+  );
+  const url = new URL(window.location.href);
+  const params = new URLSearchParams(url.search);
+  const basePath = url.pathname;
+  if (params.has('project-name')) {
+    // clear project-name query param from the url
+    params.delete('project-name');
+  }
+  const newUrl = {
+    search: `?${params.toString()}`,
+    hash: url.hash,
+  };
+  const namespacedPath = UIActions.formatNamespaceRoute(project.metadata.name, basePath, newUrl);
+
+  const handleClick = () => {
+    setLastNamespace(project.metadata.name);
```

Comment:
This (and maybe the complete handleClick should be canceled similar to #9257 when it is a ctrl+click.

```suggestion
    if (isModifiedEvent(e)) {
      return;
    }
    setLastNamespace(project.metadata.name);
```

Scenario:

When the user ctrl+clicks multiple namespaces to check their (topology) content.

Open / or /add or /topology again should not bring up the latest opened namespace, or?

Reply:
Good call on that. Just updated the code to skip setting namespace on modified clicks.